src/qt/pivx/mninfodialog.cpp

```diff
@@ -29,26 +29,30 @@ MnInfoDialog::MnInfoDialog(QWidget *parent) :
     connect(ui->pushExport, &QPushButton::clicked, [this](){ exportMN = true; accept(); });
 }
 
-void MnInfoDialog::setData(QString pubKey, QString name, QString address, QString txId, QString outputIndex, QString status)
+void MnInfoDialog::setData(const QString& _pubKey, const QString& name, QString address, const QString& _txId, const QString& outputIndex, const QString& status)
```

Comment:
since we are at it...
```suggestion
void MnInfoDialog::setData(const QString& _pubKey, const QString& name, const QString& address, const QString& _txId, const QString& outputIndex, const QString& status)
```

Reply:
done